[MTA] [RN] Release notes for MTA 8.1.1 (#347)
migtools/mta-documentation: assemblies/release-notes/assembly_mta-8-1-1.adoc

:_newdoc-version: 2.18.7
:_template-generated: 2026-04-08
:_mod-docs-content-type: ASSEMBLY

ifdef::context[:parent-context-of-mta-8-1-1: {context}]

ifndef::context[]
[id="mta-8-1-1"]
endif::[]
ifdef::context[]
[id="mta-8-1-1_{context}"]
endif::[]
= {ProductShortName} 8.1.1

:context: mta-8-1-1

[role="_abstract"]

> The {ProductShortName} 8.1.1 section lists the new features and enhancements, Technology Preview features, removed features, fixed issues, and known issues.

Author: After a complete list of RNs is confirmed, rewrite the abstract to include only the RN types included in 8.1.1. For now, there are only fixed issues.

Author: ```suggestion
The {ProductShortName} 8.1.1 section lists the issues fixed in this release.
```

Author: Confirmed by Dylan. Only fixed issues to be released -> applying the suggestion.
